windows/vtpm_cold_add_remove/handle_vm_known_issue.yml

```diff
@@ -1,15 +1,21 @@
 # Copyright 2024 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-# For Windows Server 2019/2022 VMs with SCSI disk, enabling BitLocker in guest OS returns error:
+# There are 2 known issues:
+# 1.For Windows Server 2019/2022 VMs with SCSI disk, enabling BitLocker in guest OS returns error:
 # "Set-BitLockerVolumeInternal : A device attached to the system is not functioning.
 # (Exception from HRESULT: 0x8007001F)".
-# Here is the workaround for this known issue.
+# 2.For Windows 10/11, Windows Server 2025, Bitlocker operations are slow or even may fail in guest
+# OS when VM with snapshot.
+# Here is the workaround for such issues.
 #
-- name: "Known issue - enable BitLocker failed in Windows Server 2019/2022"
+- name: "Known issue - enable BitLocker timeout/failed in {{ guest_os_name }}"
```

Comment:
timeout/failed -> timed out or failed
I didn't see the definition of `guest_os_name`. I think here can use `vm_guest_os_distribution`.

Reply:
Updated, thanks.